bridge/src/lib.rs

```diff
@@ -26,6 +27,19 @@ fn wrap_bytes(bytes: &Vec<u8>) -> AnyObject {
     RString::from_bytes(bytes, &enc).to_any_object()
 }
 
+fn to_hash_map(hash: Hash) -> HashMap<String, String> {
+    let mut result = HashMap::new();
+
+    hash.each(|k, v| {
+        result.insert(
+            k.try_convert_to::<RString>().map_err(|e| VM::raise_ex(e)).unwrap().to_string(),
+            v.try_convert_to::<RString>().map_err(|e| VM::raise_ex(e)).unwrap().to_string()
```

Comment:
nit: Doing a side effect in a `map` is I'd say not super idiomatic, and if you used `if let` instead, you'd also avoid the unwrap.